<launch>  
    <arg name="object"          default="Goesser"/>
    <arg name="path"            default="$(find learn_object)/model/$(arg object)"/>
    <arg name="filename_poses"      default="poses_circular_base.yaml"/>

    <include file="$(find camera_d435)/launch/camera_stream.launch"/>     

    <node name="learnphotos_taker" type="take_learnphotos_node.py" pkg="learn_object" output="screen">
        <param name="filename_poses"    value="$(arg path)/$(arg filename_poses)"/>
        <param name="path_object"       value="$(find learn_object)/model/$(arg object)/"/>
        <param name="object_name"       value="$(arg object)"/>
    </node> 
</launch>

